examples/linear_model/plot_sparse_logistic_regression_20newsgroups.py

```diff
@@ -18,6 +18,7 @@
 input features would be to use univariate feature selection followed by a
 traditional (l2-penalised) logistic regression model.
 """
+from __future__ import division
```

Comment:
this is one way or you just can be explicit with div so 1 / n_classes -> 1. / n_classes

link this no need to hack the pep8 rules :)